### PR TITLE
SISRP-27745 - Update jruby-openssl gem to newest version

### PR DESCRIPTION
**Pull request by davidlimcheng**
https://jira.berkeley.edu/browse/SISRP-27745

As this is my first time updating a gem, I'd like to run through what I did to ensure we're all on the same page:
* Updated `Gemfile` with newest version - this ensures we don't automatically update with each new released version.
* In ~/calcentral, ran `bundle update jruby-openssl`, followed with `gem cleanup jruby-openssl` to remove older, unused versions that don't have any dependencies.
* Would I go through the same process on each node upon merge, to update the gem on each cluster?

Additionally:
* Testing on my local machine seemed totally fine - no breaks or unexpected behavior.
* @raydavis, you mentioned getting this onto CC-QA as soon as possible - shall I open a QA PR for that upon merge of this PR?

### CODE REVIEW

**Comment by raydavis** [comment]
One happy Travis.

**Comment by raydavis** [comment]
@davidlimcheng , to avoid manual intervention on the servers, we typically update Gems by:

1. bundle update JUST_THE_MINIMUM
2. Review Gemfile.lock changes to ensure that dependencies didn't get too wildly out of control.
3. Usual testing.
4. PR for Gemfile and Gemfile.lock.

Examples: [https://github.com/ets-berkeley-edu/calcentral/commit/9048a527049d0fe879396090b9a80b88fdf998bb] , [https://github.com/ets-berkeley-edu/calcentral/commit/040ffdcfd175f11717c6421c7b0223b10219c83a]

**Comment by raydavis** [comment]
> you mentioned getting this onto CC-QA as soon as possible - shall I open a QA PR for that upon merge of this PR?

@davidlimcheng , I think that was the CC-"SIS-QA" naming problem again -- basically, getting it out on _any_ of the clusters ASAP would be reassuring. I mentioned CC-SIS-DEV & CC-"SIS-QA" because they would be the first ones to receive the update.

File: Gemfile.lock [hunk 1]
```diff
@@ -521,3 +521,6 @@ DEPENDENCIES
   torquebox-messaging (~> 3.1.2)
   torquebox-server (~> 3.1.2)
   webmock (~> 1.20.4)
+
+BUNDLED WITH
```
Comment (raydavis) [inline Gemfile.lock]: Oh, that's cute -- new mandatory boilerplate from Bundler: [http://stackoverflow.com/questions/30713922/how-to-stop-rails-from-adding-bundled-with-to-the-gemfile-lock]

Bundler is one Gem that has to installed and updated manually. Right now, all CalCentral/Junction clusters are set at Bundler version 1.9.4, which wouldn't understand this fabulous new feature. Would you be OK with deleting these new lines in the Gemfile.lock and creating a follow-up JIRA to update Bundler?

**Comment by davidlimcheng** [comment]
Thanks for the clarification, @raydavis.  I removed the extra lines added in `Gemfile.lock` and created [SISRP-28016](https://jira.berkeley.edu/browse/SISRP-28016) to update Bundler.  I don't expect it to be a very high priority at the moment, so assigned it to the next sprint - feel free to change any details or reassign the ticket.

**Comment by raydavis** [comment]
Thanks, @davidlimcheng !

**Pull request merged**